external/fv3gfs-wrapper.requirements.in

```diff
@@ -0,0 +1,10 @@
+mpi4py>=3.0.3
+cftime>=1.2.1
+xarray>=0.15.1
+netCDF4>=1.4.2
+numpy>=1.16
+pyyaml>=5
+#fv3gfs-util>=0.5.0
```

Comment:
Is this purposefully left as a comment to say something like 'in the situation where this is used, fv3gfs-util is pre installed'?

Reply:
Pretty much...yes.

Reply:
Could you just write that in the comment then, so it's more direct?